docs/sphinx/about/whats-new.txt

```diff
@@ -4,7 +4,8 @@ Version history
 5.1.5 (2015 October 12)
 -----------------------
 
-* Bug fixes, including:
+* Java changes:
+   - Bug fixes, including:
```

Comment:
The indentation on this line does not make sense anymore since the list below are bug fixes. Can you recombine it with the line above?